rockstar/RockStar.py

```diff
@@ -10,21 +10,22 @@
 import click
 import git
 
-hello_world_c = """#include <iostream>
+HELLO_WORLD_C = """#include <iostream>
 int main()
 {
   std::cout << "Hello World!" << std::endl;
   return 0;
 }
 """
 
-default_file_name = 'main.cpp'
+DEFAULT_FILE_NAME = 'main.cpp'
 
 
 class RockStar:
 
-    def __init__(self, days=400, days_off=[], file_name=default_file_name,
-                 code=hello_world_c, off_fraction=0.0):
+    def __init__(self, days=400, days_off=(), file_name=DEFAULT_FILE_NAME,
+                 code=HELLO_WORLD_C, off_fraction=0.0):
+        self.repo = None
```

Comment:
It's better to define all the instance attributes within `__init__ ` for improving readability. Please refer to [this](http://stackoverflow.com/questions/19284857/instance-attribute-attribute-name-defined-outside-init) link.